internal/driver/commands.go

```diff
@@ -199,6 +199,14 @@ var configHelp = map[string]string{
 		"Use name=value syntax to limit the matching to a specific tag.",
 		"Numeric tag filter examples: 1kb, 1kb:10kb, memory=32mb:",
 		"String tag filter examples: foo, foo.*bar, mytag=foo.*bar"),
+	"tagleaf": helpText(
+		"Adds a pseudo leaf location for samples with a matching tag.",
+		"Use name=value syntax to specify the tag and the value for which nodes would be added.",
```

Comment:
Can we make sure and clarify that the same syntax as for tagfocus / tagignore can be used for values? E.g. numeric values with units, range and so on. And would be good to reflect this in the examples, like the documentation for other tag filters does.

Reply:
I have updated the description.  There is difference between tagfocus/tagignore syntax and the one used for tagroot/tagleaf (as I understand).  tagroot and tagleaf add locations when specific tags exist in a sample, there values are immaterial (though they would appear in the added location).